### PR TITLE
Fix readability/typo/pep8 issues

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch fixes some typos, pep8 issues. Also Now
the context will have type "-" object_name for better
readability

tendrl-bug-id: Tendrl/gluster-integration#361
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

File: tendrl/gluster_integration/message/process_events.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def process_events():
                 # this perticular event is recovery event
                 # so process this event and delete it
                 event_utils.emit_event(
-                    event.context.split("_")[0],
+                    event.context.split("-")[0],
```
Comment (shtripat) [inline tendrl/gluster_integration/message/process_events.py]: Is it good to use hyphon as separator? What are changes are having hyphon in data parts itself. I feel rather we should pipe may be

Comment (nnDarshan) [reply tendrl/gluster_integration/message/process_events.py]: ack. Good suggestion. Using `|` now

**Comment by nnDarshan** [comment]
@r0h4n Pls review

**Pull request merged**